tests/func/test_data_cloud.py

```diff
@@ -164,7 +164,7 @@ def get_local_url():
 
 
 def get_ssh_url():
-    return "ssh://{}@127.0.0.1:{}".format(
+    return "ssh://{}@127.0.0.1:22{}".format(
```

Comment:
Is this really needed? 22 should be used by default already.

Reply:
This is needed as long as we compare urls as strings. It was easier for me to change it here, than change a test.

Ultimately, we need a proper path refactor, which will solve all such inconsistencies. I will try to add it in a separate PR tomorrow.